app/components/works/embargo_component.html.erb

```diff
@@ -40,7 +40,11 @@
   <div class="mb-3 row">
     <%= form.label :access, 'Who can access?', class: 'col-sm-2 col-form-label' %>
     <div class="col-sm-10">
-      <%= form.select :access, [['Stanford Community', 'stanford'], ['Everyone', 'world']], { disabled: "", prompt: "Select..." }, class: "form-select", required: true %>
+      <% if default_access == 'depositor-selects' %>
```

Comment:
Can you move this conditional into the class?  So that we have `depositor_selects_access?`